configs/ruby/GPU_VIPER.py

```diff
@@ -492,6 +494,15 @@ def define_options(parser):
     parser.add_argument(
         "--glc-atomic-latency", type=int, default=1, help="GLC Atomic Latency"
     )
+    parser.add_argument(
+        "--atomic-alu-latency", type=int, default=0, help="Atomic ALU Latency"
```

Comment:
Why is this default different from the default in RubyCache.py?

Reply:
Our idea was that while the GPU currently only uses these parameters, we wanted to set GPU specific parameters/values in VIPER and not set the same values by default for anyone else trying to use these parameters for other code.  But if desired we can certainly make them the same given that no one else uses it as is.

Reply:
Whatever makes sense to reduce user confusion is good for me. Usually having multiple places with different defaults adds to confusion.

Reply:
I've updated the defaults in RubyCache.py to match